lib/components/narrative/trip-tools.js

```diff
@@ -55,6 +56,12 @@ class CopyUrlButton extends Component {
   render() {
     return (
       <div>
+        {/* Announces copy button status to AT */}
+        <InvisibleA11yLabel aria-live="assertive">
+          {this.state.showCopied && (
+            <FormattedMessage id="components.TripTools.linkCopied" />
```

Comment:
The conditional should probably include the label too right

Reply:
Actually, conditionally rendering the `aria-live` causes inconsistent announcements on AT. The `aria-live` `div` needs to be present even when it's empty, its like a tiny status region

Reply:
Yikes that is UGLY! The a11y spec really has a ways to go...

Reply:
Long term we may want to look at creating a global status region that handles little announcements like this as well as the results from the geocoder, trip results, etc!!